<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadDialog</class>
 <widget class="QDialog" name="LoadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Files</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Batch &amp;Folder:</string>
          </property>
          <property name="buddy">
           <cstring>importFolder</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="importFolder"/>
        </item>
        <item>
         <widget class="QPushButton" name="browseBtn">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="copyFiles">
          <property name="text">
           <string>&amp;Copy images to folder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="copyToFolder">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="copyBrowseBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="removeFlag">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Remove images &amp;after copying</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Performance</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Preload Memory Usage:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="horizontalSlider">
        <property name="minimum">
         <number>256</number>
        </property>
        <property name="maximum">
         <number>4069</number>
        </property>
        <property name="singleStep">
         <number>256</number>
        </property>
        <property name="pageStep">
         <number>512</number>
        </property>
        <property name="value">
         <number>1536</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Look Ahead:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSlider" name="horizontalSlider_2">
        <property name="maximum">
         <number>16</number>
        </property>
        <property name="value">
         <number>12</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="lookAhead">
        <property name="suffix">
         <string> images</string>
        </property>
        <property name="maximum">
         <number>16</number>
        </property>
        <property name="value">
         <number>12</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Look Behind:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSlider" name="horizontalSlider_3">
        <property name="maximum">
         <number>16</number>
        </property>
        <property name="value">
         <number>4</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QSpinBox" name="lookBehind">
        <property name="suffix">
         <string> images</string>
        </property>
        <property name="maximum">
         <number>16</number>
        </property>
        <property name="value">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="cacheSize">
        <property name="suffix">
         <string> MB</string>
        </property>
        <property name="minimum">
         <number>256</number>
        </property>
        <property name="maximum">
         <number>4066</number>
        </property>
        <property name="singleStep">
         <number>256</number>
        </property>
        <property name="value">
         <number>1536</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LoadDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>427</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LoadDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>495</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>copyFiles</sender>
   <signal>toggled(bool)</signal>
   <receiver>copyToFolder</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>copyFiles</sender>
   <signal>toggled(bool)</signal>
   <receiver>copyBrowseBtn</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>506</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>copyFiles</sender>
   <signal>toggled(bool)</signal>
   <receiver>removeFlag</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
